sklearn/utils/__init__.py

```diff
@@ -114,6 +114,12 @@ def _in_unstable_openblas_configuration():
     return False
 
 
+@validate_params(
+    {
+        "X": ["array-like", "sparse matrix"],
+        "mask": ["array-like", np.ndarray],
```

Comment:
`ndarray` are included in `"array-like"`:

```suggestion
        "mask": ["array-like"],
```

Could you update the documentation of `mask` by stating `array-like` instead of `ndarray`.

Reply:
@glemaitre working on it. Will be submitting changes soon.